src/vue/pages/Movements.vue

```diff
@@ -21,14 +21,14 @@
         class="movements__actions"
         slot="extra"
       >
-        <button
+        <!-- <button
           v-ripple
           class="app__button-raised movements__button-raised"
           @click="isWithdrawalDrawerShown = true"
         >
           <i class="mdi mdi-download movements__btn-icon" />
           {{ 'op-pages.withdrawal' | globalize }}
-        </button>
+        </button> -->
```

Comment:
these comment should be resolved when https://github.com/tokend/web-client/pull/136 merged